src/services/lock/repository/lock-node.repository.ts

```diff
@@ -10,7 +10,7 @@ export class LockNodeRepository {
   ) {}
 
   async distinct(field: keyof LockNode) {
-    return await this.lockNodeModel.distinct(field);
+    return await this.lockNodeModel.distinct<LockNode>(field);
```

Comment:
Interesting, you'd think this would be inferred

Reply:
Does setting the return type of the function to `Promise<LockNode>` work? I think that more indicates what you are trying to get across.